languages/csharp-dotnet-2.2/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "artifacts": [
+        {"source":"/opt/algorithm/bin/Release/netcoreapp2.2/__ALGO__.dll", "destination":"/opt/algorithm/pipe.dll"},
+        {"source":"/opt/algorithm/bin/Release/netcoreapp2.2/__ALGO__.runtimeconfig.json", "destination": "/opt/algorithm/pipe.runtimeconfig.json"},
```

Comment:
Turns out with dotnet, the only components required to actually execute a compiled algorithm are the `dll` and `runtimeconfig.json` files, assuming the environment remains the same.

TODO: I need to test this.